src/pages/programs/program-solution-entity/program-solution-entity.html

```diff
@@ -3,20 +3,15 @@
 </ion-header>
```

Comment:
This page is changed every where for submissionId and submissionStatus to _id and status